resources/metadata/abapEnvironmentCreateSystem.yaml

```
@@ -179,6 +179,46 @@ spec:
           - STAGES
           - STEPS
           - GENERAL
+      - name: abapSystemParentServiceLabel
+        description: The name of the parent service label that is passed through to CLD
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: abapSystemParentServiceInstanceGuid
```

UUID, HTTPS etc are all in capital letters -> imo, GUID falls in the same category. What do you think?
```suggestion
      - name: abapSystemParentServiceInstanceGUID
```

---

True. When implementing I looked for similar appearances of `Guid`. Seems like I was wrong... Changed it